Makefile

```diff
@@ -16,8 +17,15 @@ all: build
 update: build push
 
 build: ## Build docker image with name LOCAL_IMAGE (NAME:RELEASE_VERSION).
-	docker build -f $(THISDIR_PATH)/Dockerfile -t $(LOCAL_IMAGE) $(PROJECT_PATH) \
-		--build-arg RELEASE_VERSION=$(RELEASE_VERSION)
+	@if [ -f "./front-end/quickstart/target/api-design-studio-${RELEASE_VERSION}-quickstart.zip" ] || [ ${LOCAL_RELEASE} == "true" ]; then \
```

Comment:
I Could set the RELEASE_PATH variable in the subshell, but doing it replicating the `docker build` command it improves readability